src/lib/y2users/config.rb

```diff
@@ -125,6 +126,11 @@ def merge!(other)
       self
     end
 
+    # Useradd configuration to be applied to the system before creating the users
+    #
+    # @return [UseraddConfig, nil] nil if the configuration is unknown
+    attr_accessor :useradd_config
```

Comment:
I think we need to copy/clone this object when doing a deep copy with `#copy` method.

Reply:
And what happens when merging configs?